api-errors.json

```
@@ -643,6 +643,10 @@
         "short_message": "Unavailable For Legal Reasons",
         "long_message": "Unavailable For Legal Reasons"
     },
+    "49900": {
+        "short_message": "Client Closed Request",
+        "long_message": "Client Closed Request"
```

Maybe have a longer description here e.g.
```suggestion
        "long_message": "Client closed the connection before the server answered the request"
```

---

Updated